docs/dev/version-migration/v0.1-to-v0.2-migration.md

```diff
@@ -0,0 +1,168 @@
+# v0.2.* (latest v0.2.7)
+
+* New features
+    * Adds support for `DISTINCT`, `LET` (from `FROM` clause), system stored procedure calls (`EXEC`)
+    * Adds parser support for DML statements (`INSERT`, `UPDATE`, `DELETE`), `Parameter`, `BY` variable
+    * Improvements to `LIKE` pattern compilation performance
+    * Adds function to convert from UNIX epoch to TIMESTAMP and TIMESTAMP to UNIX epoch
+* Deprecated items
+    * `AstRewriter`, `AstRewriterBase`, `MetaStrippingRewriter`, `RewriterTestBase`
+        * Existing AST rewriters migrated to use PIG’s `VisitorTransform`s
+        * Migration guide provided [here](https://github.com/partiql/partiql-lang-kotlin/blob/feb84730c64a2ad0f12c57bef3b1c45e21279538/docs/dev/RewriterToVisitorTransformGuide.md)
+* Misc/bug fixes
+    * New error codes for division by `0` and modulo `0`
+    * [fix] float negative zero equality
+    * Removes invalid syntax check on case expressions w/ type parameters e.g., `CAST(a AS DECIMAL(1, 2))` now does not throw
+    * [fix] `LIMIT` with value over 2^31 returning no values
+    * [fix] `LIMIT` clause execution order
+    * [fix] stop treating date parts as if they are string literals
+    * [fix] parsing of `TRIM` specification keywords (`BOTH`, `LEADING`, and `TRAILING`)
+    * Adds some AST rewriters — `SelectStarRewriter` and `StaticTypeRewriter`
+## Breaking changes
+### Breaking behavioral changes
+1. `JOIN` requires an `ON` clause. In v0.1.*, the `ON` clause was optional, which caused ambiguous parsing of multiple `JOIN`
+```kotlin
+// v0.1.*
+// Initialization of components related to evaluating a query end-to-end
+val ion = IonSystemBuilder.standard().build()
+val parser = SqlParser(ion)
+val pipeline = CompilerPipeline.standard(ion)
+val evaluationSession = EvaluationSession.standard()
```

Comment:
Can consolidate some of the boilerplate that's used by both versions. Will update in upcoming revision

Reply:
Is this resolved? Hard for me to tell without prior review

Reply:
Yes, sorry should have marked the comment as resolved. I removed some of the boilerplate in the code examples in https://github.com/partiql/partiql-lang-kotlin/pull/688/commits/aabbc98dc3097d601b2e4d9bcb083d66cf9a576a.